examples/classifier_compression/compress_classifier.py

```diff
@@ -99,7 +99,11 @@ def load_test_data(args):
                                                 args.dataset, add_softmax=True, verbose=False)
         do_exit = True
     elif args.qe_calibration:
```

Comment:
The way you left it, if someone does `--qe-calibration --quantize-eval` it'll still exit after the calibration. That's not what you meant, was it?
I think it should be something like this, but double check me:

```suggestion
     elif args.qe_calibration and not (args.evaluate and args.quantize_eval):
```